config/ETH_ddpm.yml

```diff
@@ -0,0 +1,67 @@
+DATA_AGGREGATION:
+  TXT_DATA_DIR: '/home/bryan/Documentos/Seminario/coolName2/txtFiles/'
+  CSV_DATA_DIR: '/home/bryan/Documentos/Seminario/coolName2/csvFiles/'
+  RAW_DATA_DIR: '/home/bryan/Documentos/Seminario/coolName2/preprocessCsv/'
+  AGG_DATA_DIR: '/home/bryan/Documentos/Seminario/coolName2/preprocessCsv/'
+PICKLE:
+  PICKLE_DIR: '/home/est_licenciatura_bryan.calderon/coolName2/pickle/'
```

Comment:
wrong

Reply:
@Ethiqque you've commented an outdated version of this file.
A similar comment was done and it was already addressed.